test/parallel/test-fs-truncate.js

```diff
@@ -190,6 +190,31 @@ function testFtruncate(cb) {
     );
   });
 
+  [-1.5, 1.5].forEach((input) => {
+    assert.throws(
```

Comment:
It's better to use [`common.expectsError`](https://github.com/nodejs/node/blob/7d06761f839bbff8c671b0e1a6d541faa5452354/test/common/README.md#expectserrorfn-settings-exact) in our tests.

Reply:
Updated. thanks for review :)

Reply:
@starkwang I must disagree. The original reason for using `common.expectsError` is that it allowed to use the error object. Now, `assert.throws` is more powerful than `common.expectsError` due to a much better error message output and since that feature was added to `assert.throws`.